FWCore/Framework/interface/Callback.h

```diff
@@ -47,12 +48,12 @@ namespace edm {
                              =CallbackSimpleDecorator<TRecord> >
       class Callback : public CallbackBase {
        public:
-         typedef TReturn (T ::* method_type)(const TRecord&);
+         using  method_type = TReturn (T ::*)(const TRecord&);
```

Comment:
@Dr15Jones for my own education, is there a functional difference between these two syntaxes, or is it just aesthetics? Browsing around https://en.cppreference.com/w/cpp/language/type_alias I would say the latter, but I would like to know your motivation

Reply:
The `using` directive can do everything the `typedef` can do and then even more. The modern C++ community seems to favor deprecating the use of `typedef` in favor of the more flexible `using`. That is why I've been converting code from using one to the other.